src/Helpers/ElementHelpers/ElementHelper.cs

```diff
@@ -203,49 +203,71 @@ public List<PageSummary> GenerateQuestionAndAnswersList(string guid, FormSchema
 
             foreach (var page in formSchema.Pages.ToList())
             {
+                var formSchemaQuestions = new List<IElement>();
+
+                if (page.Elements.Any(_ => _.Type == EElementType.AddAnother))
+                {
+                    var listOfPageSummary = new List<PageSummary>();
+                    var addAnotherElement = page.Elements.FirstOrDefault(_ => _.Type == EElementType.AddAnother);
+                    var currentIncrement = GetAddAnotherNumberOfFieldsets(addAnotherElement, formAnswers);
+                    for (var i = 1; i <= currentIncrement; i++)
+                    {
+                        var addAnotherPageSummary = new PageSummary
+                        {
+                            PageTitle = addAnotherElement.GetLabelText(page.Title),
+                            PageSlug = $"{page.PageSlug}-{addAnotherElement.Properties.QuestionId}-{i}"

```

Comment:
the change link should just be pageSlug, assuming the change url would not work as it has the questionID within?